### PR TITLE
Update footer to match prototype

### PR DESCRIPTION
**Pull request by paulodeon**
### Context

Give more help to users of the service

### Changes proposed in this pull request
The "Get Help" section here: https://refer-misconduct-prototype.herokuapp.com/

### Link to Trello card

[](https://trello.com/c/84NFudI7/1169-update-the-footer)

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: app/views/layouts/base.html.erb [hunk 1]
```diff
@@ -41,12 +41,38 @@
...
+            <path fill="currentColor" d="M421.5 142.8V.1l-50.7 32.3v161.1h112.4v-50.7zm-122.3-9.6A47.12 47.12 0 0 1 221 97.8c0-26 21.1-47.1 47.1-47.1 16.7 0 31.4 8.7 39.7 21.8l42.7-27.2A97.63 97.63 0 0 0 268.1 0c-36.5 0-68.3 20.1-85.1 49.7A98 98 0 0 0 97.8 0C43.9 0 0 43.9 0 97.8s43.9 97.8 97.8 97.8c36.5 0 68.3-20.1 85.1-49.7a97.76 97.76 0 0 0 149.6 25.4l19.4 22.2h3v-87.8h-80l24.3 27.5zM97.8 145c-26 0-47.1-21.1-47.1-47.1s21.1-47.1 47.1-47.1 47.2 21 47.2 47S123.8 145 97.8 145"></path>
+          </svg>
+
+          <span class="govuk-footer__licence-description">All content is available under the <a class="govuk-footer__link" href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/">Open Government Licence v3.0</a>, except where otherwise stated</span>
+        </div>
+
+        <div class="govuk-footer__meta-item"><a class="govuk-footer__link govuk-footer__copyright-logo" href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">© Crown copyright</a></div>
```
Comment (vassyz) [inline app/views/layouts/base.html.erb:74]: You can do this if you want to:

```
<div class="govuk-footer__meta-item">
  <%= footer.copyright %>
</div>
```

Comment (paulodeon) [reply app/views/layouts/base.html.erb:74]: Niiice

File: app/views/layouts/base.html.erb [hunk 2]
```diff
@@ -41,12 +41,40 @@
...
-      meta_items: {
-        'Accessibility': '/accessibility',
-        'Cookies': '/cookies',
-        'Privacy notice': '/privacy'
-      }) %>
+    <%= render GovukComponent::FooterComponent.new do |footer| %>
+      <% footer.with_meta do %>
+        <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+          <h2 class="govuk-heading-m">Get help</h2>
+          <p class="govuk-!-font-size-16">Email: <a class="govuk-footer__link" href="mailto:misconduct.teacher@education.gov.uk">misconduct.teacher@education.gov.uk</a><br>You’ll get a response within 3 working days.</p>
```
Comment (tavy88) [inline app/views/layouts/base.html.erb:48]: We could use `link_to` here.

File: app/views/layouts/base.html.erb [hunk 3]
```diff
@@ -41,12 +41,40 @@
...
+
+          <ul class="govuk-footer__inline-list">
+            <li class="govuk-footer__inline-list-item">
+              <%= govuk_link_to("Accessibility", "/accessibility", class: "govuk-footer__link") %>
+            </li>
+            <li class="govuk-footer__inline-list-item">
+              <%= govuk_link_to("Cookies", "/cookies", class: "govuk-footer__link") %>
+            </li>
+            <li class="govuk-footer__inline-list-item">
+              <%= govuk_link_to("Privacy notice", "/privacy", class: "govuk-footer__link") %>
```
Comment (malcolmbaig) [inline app/views/layouts/base.html.erb:63]: Worth using the Rails path helpers for these links (eg— `cookies_path`).

**Review by malcolmbaig — APPROVED** [review]

**Pull request merged**